scipy/signal/_splinemodule.h

```diff
@@ -1,25 +1,339 @@
-#include "Python.h"
-#include "numpy/arrayobject.h"
-#include <math.h>
+#pragma once
+#include"Python.h"
```

Comment:
Missing space (162ed2d602240bb291b0a927ea7f94426789b36b)

(leaving commit I'm commenting on in comment to make it easier to squash to the right commit; if you use `git --fixup <hash>`, rebasing will do it for you automatically)